pkg/addons/data/ConfigMap/openshift-metrics/prometheus.yaml

```diff
@@ -16,66 +16,6 @@ data:
     # Kubernetes labels will be added as Prometheus labels on metrics via the
     # `labelmap` relabeling action.
 
-    # Scrape config for API servers.
```

Comment:
Why is this removed?

Reply:
Do we want to expose their api server and controller metrics to users? As they dont see them anywhere should they be concern on metrics of them? "metrics from the other side" ?

Reply:
We need to be gathering control plane metrics and the prometheus installed by us is for cluster monitoring. I think @pweil- and @jim-minter felt users need to deploy their own prometheus server for their own apps but I still haven't seen multitenant prom deployments to know how that will look like.

Reply:
I was under impression that this Prometheus will be "user one". For our control plain, we might need to have something like https://github.com/weaveworks/cortex . We need metrics, but this goes to the same conversation pool, "where to store etcd backups" kinda talks.